test/remote/gateways/remote_payu_latam_test.rb

```diff
@@ -146,11 +146,11 @@ def test_successful_purchase_colombia
         zip: "01019-030",
         phone: "(11)756312633"
       ),
-      tx_tax: '3193',
-      tx_tax_return_base: '16806'
+      tax: "3193",
+      tax_return_base: "16806"
     }
 
-    response = gateway.purchase(@amount, @credit_card, @options.update(options_colombia))
+    response = gateway.purchase(2000000, @credit_card, @options.update(options_colombia))
```

Comment:
What did this alter?

Reply:
You cant use a value less than the tax values. So I had to increase the purchase amount for the test to pass.

Reply:
Would it make sense to just bump up `@amount` then?

Reply:
Just tried that and it fails another test.